wallet/test/Test/Pos/Wallet/Web/AddressSpec.hs

```diff
@@ -0,0 +1,71 @@
+module Test.Pos.Wallet.Web.AddressSpec
+       ( spec
+       ) where
+
+import           Universum
+
+import           Data.Default                 (def)
+import           Formatting                   (sformat, (%))
+import           Serokell.Data.Memory.Units   (memory)
+import           Test.Hspec                   (Spec, describe)
+import           Test.Hspec.QuickCheck        (prop)
+import           Test.QuickCheck              (arbitrary)
+import           Test.QuickCheck.Monadic      (pick)
+
+import           Pos.Binary                   (biSize)
+import           Pos.Client.Txp.Addresses     (getFakeChangeAddress, getNewAddress)
+import           Pos.Core.Address             (Address)
+import           Pos.Crypto                   (PassPhrase)
+import           Pos.Launcher                 (HasConfigurations)
+import           Pos.Util.CompileInfo         (withCompileInfo)
+
+import           Pos.Util.CompileInfo         (HasCompileInfo)
+import           Pos.Wallet.Web.Account       (GenSeed (..), genUniqueAddress)
+import           Pos.Wallet.Web.ClientTypes   (AccountId, CAccountInit (..), caId, cwamId)
+import           Pos.Wallet.Web.Methods.Logic (newAccount)
+import           Pos.Wallet.Web.State         (getWalletAddresses)
+import           Pos.Wallet.Web.Util          (decodeCTypeOrFail)
+import           Test.Pos.Util                (assertProperty, expectedOne,
+                                               withDefConfigurations)
+import           Test.Pos.Wallet.Web.Mode     (WalletProperty)
+import           Test.Pos.Wallet.Web.Util     (importSingleWallet, mostlyEmptyPassphrases)
+
+spec :: Spec
+spec = withCompileInfo def $
+       withDefConfigurations $
+    describe "Fake address had maximal possible size" $ do
```

Comment:
had?

Reply:
Lol mda.
Thanks for your thorough review